publications/ohbm_2024/brainhack_project.md

```diff
@@ -0,0 +1,112 @@
+To be submitted [here](https://github.com/ohbm/hackathon2024/issues/new?assignees=sina-mansour&labels=Hackathon+Project&projects=&template=brainhack-hacktrack-project.yml&title=%3CMy+Project+Name%3E)
+
+Combined from previous years:
+- 2023: https://github.com/ohbm/hackathon2023/issues/15
+- 2022: https://github.com/ohbm/hackathon2022/issues/48
+
+# Title
+(Name of your awesome project. Please also update the title of the issue to be the title of your project)
+```txt
+Nilearn: Statistics and Machine Learning for Neuroimaging in Python
+```
```

Comment:
In case it was unclear earlier: The answers are inside the these '''txt ... ''' blocks.